src/topic-client.ts

```diff
@@ -45,17 +45,17 @@ export class TopicClient {
    *
    * @param {string} cacheName - The name of the cache to containing the topic to subscribe to.
    * @param {string} topicName - The name of the topic to subscribe to.
-   * @param {SubscribeCallOptions} options - The options for the subscription.
-   * @param {function} options.onItem - The callback to invoke when data is received.
-   * @param {function} options.onError - The callback to invoke when an error is received.
+   * @param {SubscribeCallOptions} options - The options for the subscription. Defaults to no-op handlers.
+   * @param {function} options.onItem - The callback to invoke when data is received. Defaults to no-op.
+   * @param {function} options.onError - The callback to invoke when an error is received. Defaults to no-op.
    * @returns {Promise<TopicSubscribe.Response>} -
    * {@link TopicSubscribe.Subscription} on success.
    * {@link TopicSubscribe.Error} on failure.
    */
   public async subscribe(
     cacheName: string,
     topicName: string,
-    options: SubscribeCallOptions
+    options: SubscribeCallOptions = {}
```

Comment:
is there a use case where a user can do something useful with a call to subscribe without passing one of the handlers?  I think I might keep this option as required, thoughts?

Reply:
That's reasonable!